cross/ser2net/Makefile

```diff
@@ -0,0 +1,21 @@
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+PKG_NAME = ser2net
```

Comment:
I looks like this file is not related to this PR

Reply:
You are absolutely correct, thanks for spotting this.